packages/utils/src/lib/reports/md/table.ts

```diff
@@ -40,13 +40,14 @@ export function tableHtml(data: (string | number)[][]): string {
     throw new Error("Data can't be empty");
   }
 
+  // @TODO add formatting via prettier
   const tableContent = data.map((arr, index) => {
     if (index === 0) {
-      const headerRow = arr.map(s => `<th>${s}</th>`).join('');
-      return `<tr>${headerRow}</tr>`;
+      const headerRow = arr.map(s => `<th>${s}</th>\n`).join('');
```

Comment:
❔ 👨‍💻 Consistency: I see `\n` and `NEW_LINE` being used in some places for the reports. Is there a system?

Reply:
NO! I try to tackle this as good as possible in this PR!
Thx for pointing it out too! <3

Reply:
Now there is a system.